src/eXpansion/Bundle/CustomChat/Plugins/CustomChat.php

```diff
@@ -129,16 +129,27 @@ public function onPlayerChat(Player $player, $text)
 
     /**
      * @param Player $player
-     * @param $text
-     * @param $color
-     * @param null $group
+     * @param        $text
+     * @param        $color
+     * @param null   $group
      */
     private function sendChat(Player $player, $text, $color, $group = null)
     {
         $nick = trim($player->getNickName());
         $nick = str_ireplace('$w', '', $nick);
         $nick = str_ireplace('$z', '$z$s', $nick);
-        $replacements = [":yes:" => "",":no:" => "", ":happy:" => "", ":sad:" => "", ":heart:" => ""];
+        $replacements = [
```

Comment:
Can't we have this as a parameter ? in the yml ?

Reply:
yes we can have it :)